src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/**
+* Represents the static data of a user.
+*/
+public class User {
+    private final String userID;
+    private final String username;
+    private final String firstName;
+    private final String lastName;
+    private final String profilePicture;
+    private final ArrayList<String> gamesCreated;
+    private final int numGamesFinished;
+    private final int numGamesCreated;
+    private final ArrayList<Pair<String, Long>> gamesCompletedWithTime;
+
+    /**
+    * Builder class used to create User objects.
+    */
+    public static class Builder {
+        // Required parameters
+        private final String userID;
+
+        // Optional parameters - initialized to default values
+        private String username = null;
+        private String firstName = null;
+        private String lastName = null;
+        private String profilePicture = null;
+        private ArrayList<String> gamesCreated = null;
+        private int numGamesFinished = -1;
+        private int numGamesCreated = -1;
+        private ArrayList<Pair<String, Long>> gamesCompletedWithTime = null;
```

Comment:
Why don't you use a Map?

Reply:
This allows us to store two or more different times for the same game. Also, I don't think we'll ever have to look up a time for a specific game.

Reply:
You can use Multimap (https://guava.dev/releases/23.0/api/docs/com/google/common/collect/Multimap.html).

Reply:
I just don't see the purpose of using a map here. Every time we access gamesCompletedWithTime, we're going to iterate through the entire list anyways. So the fast lookup speed of Map/Multimap, which is its main benefit, isn't going to be used at all.

Reply:
Do you plan to use the gamesCompletedWithTime to calculate the average time per game or the best play time per game? If this is the case, then you shouldn't store all raw <gameId, completedTime>. Because the list will keep growing and waste storage. Imagine that the wasted storage is huge if you have x10 million users. For the user statics purpose, try to aggregate the data and only store the necessary data (e.g. the average time or the best time for playing the game).

Reply:
The plan is to create a histogram with game difficulty on the x axis and time on the y axis. Also, I prefer storing all the raw data just in case we want to do something different with it later on. We don't expect this to be a problem because we don't expect to have 10 million users. :P